ngraph/frontend/onnx/frontend/src/frontend.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <frontend_manager/frontend_exceptions.hpp>
+#include <frontend_manager/frontend_manager.hpp>
+#include <onnx_frontend/frontend.hpp>
+#include <onnx_frontend/input_model.hpp>
+#include <onnx_import/onnx.hpp>
+
+using namespace ngraph;
+using namespace ngraph::frontend;
+
+extern "C" FRONTEND_API FrontEndVersion GetAPIVersion()
+{
+    return OV_FRONTEND_API_VERSION;
+}
+
+extern "C" FRONTEND_API void* GetFrontEndData()
+{
+    FrontEndPluginInfo* res = new FrontEndPluginInfo();
+    res->m_name = "onnx";
```

Comment:
My proposal is to change this name to 'onnx_experimental' so that MO will not try to use it for existing 'onnx' model conversion.
And also it will be possible to enable this by default in CI, and remove 'onnx_reader' from build